.github/workflows/nightly-clickbench.yml

```diff
@@ -0,0 +1,166 @@
+name: Nightly-Clickbench-Test
+
+on:
+  schedule:
+    - cron: '0 5 * * *'  # Every day at midnight EST (UTC-5)
+  workflow_dispatch:  # Allows manual trigger as well
+
+jobs:
+  Clickbench-10M-Test:
+    timeout-minutes: 1440
```

Comment:
we should do 8-10 hours. so that it does not fall into the next run

Reply:
This wouldn't fall into the next run since, jobs would be scheduled for execution. I have kept it to 1 day for safe measure. It would probably be better to keep a larger buffer.

Reply:
addressed in the latest commit.